packages/form/src/hook-form/TextArea.tsx

```diff
@@ -14,7 +13,7 @@ interface TextAreaProps {
   disabled?: boolean;
 }
 
-const TextArea = ({ label, name, validators, helptext, textareaClass, id, disabled }: TextAreaProps): JSX.Element => {
```

Comment:
Litt rar endring som ein del av bumping syns eg, men reknar med det berre vart oppdaga at denne property ikkje er i bruk?

Reply:
Denne propertien var den som bruktes for å rendre importen som er fjernet. Propen var aldri i bruk og dermed trenger vi ikke erstatte komponenten som skulle fases ut

Reply:
`@navikt/ft-plattform-komponenter` eskporterer ikke lenger `ExpandableLabel`